app/src/main/java/com/example/androidacademyhw/movies/FragmentMoviesList.kt

```diff
@@ -3,31 +3,27 @@ package com.example.androidacademyhw.movies
 import android.content.Context
 import android.os.Bundle
 import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import com.example.androidacademyhw.FragmentClickListener
 import com.example.androidacademyhw.R
 import com.example.androidacademyhw.Tags
-import com.example.androidacademyhw.data.Actor
-import com.example.androidacademyhw.data.Genre
 import com.example.androidacademyhw.data.Movie
-import com.example.androidacademyhw.data.loadMovies
 import com.example.androidacademyhw.databinding.FragmentMoviesListBinding
 import com.example.androidacademyhw.details.FragmentMoviesDetails
-import kotlinx.coroutines.*
+import com.example.androidacademyhw.movies.MoviesAdapter.OnMovieClickListener
 
-
-class FragmentMoviesList : Fragment() {
+class FragmentMoviesList : Fragment(), OnMovieClickListener {
 
     private var fragmentMoviesClickListener: FragmentClickListener? = null
 
     private var _binding: FragmentMoviesListBinding? = null
     private val binding get() = _binding!!
```

Comment:
it is not a safe way to use !!, but for now it is okay.

Reply:
I'll keep that in mind. Thanks